src/popup/components/AccountCardMultisig.vue

```diff
@@ -11,7 +11,10 @@
     </template>
 
     <template #middle>
-      <BalanceInfo :balance="+account.balance" />
+      <BalanceInfo
+        :balance="+account.balance"
+        :protocol="PROTOCOL_AETERNITY"
```

Comment:
```suggestion
        :protocol="account.protocol"
```

Reply:
There should be PROTOCOL_AETERNITY selected as multisig views are using AE protocol